Expensify/App
# Add Welcome Screen

### Details

- Adds a Welcome Screen step for the first login of the user after setting the password
- User has an option to upload Avatar, update name and add secondary login
- User also has an option to skip the step completely

### Fixed Issues

$ https://github.com/Expensify/App/issues/5079

### Tests
1. Tested with a fresh account creation
2. Tested with skipping the step
3. Tested with an unvalidated account with validation link expired
4. Tested an existing account Forgot password flow




### QA Steps

1. Visit the app as Guest
2. Enter a new email address as a fresh signup
3. Copy the magic link received on the email and hit the URL
4. Enter the password in the form and click on Submit
5. You'll be shown the Welcome step form
6. Fill the details and verify in the Profile page that all the data is stored.

### Tested On


### Screenshots


#### Web

<img width="605" alt="Screenshot 2022-01-03 at 5 04 36 AM" src="https://user-images.githubusercontent.com/57435789/147892307-47845d97-84f5-444a-8f25-039a762fd22a.png">


#### Mobile Web

![image](https://user-images.githubusercontent.com/57435789/147892547-af65917d-0c71-49f0-aa4e-7fa8b04f674b.png)

#### Desktop


#### iOS


#### Android
